collins/ipam.go

```diff
@@ -149,6 +149,33 @@ func (s IPAMService) Pools() ([]Pool, *Response, error) {
 	return c.Pools, resp, nil
 }
 
+// IPAMService.IpmiPools return a list of the available IP pools for IPMI/OOB
+// networks.
+//
+// http://tumblr.github.io/collins/api.html#api-ipam-get-ipmi-address-pools
+func (s IPAMService) IpmiPools() ([]Pool, *Response, error) {
```

Comment:
Given that `IPMI` is an acronym, the function signature should reflect that. Basically, upcase `Ipmi`.

Reply:
That would for sure be the idiomatic golang way to do this. I didn't want to cause any breakage though for consumers of this client since this is just being moved out of the ipmi.go file.

Reply:
I think we can change this to `IPMIPools()`. We have marked the library as `v0` on gopkg after all, and I don't think there are that many downstream consumers of this feature yet.

Speaking of which, maybe it is time to go to `v1`? The API has been mostly stable for quite a while now.

Reply:
Sounds good to me, I updated the method and tests with this change.